File: LeetCode/Python/Easy/242-Valid-Anagram/solution.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        s_frequencies: list[int]=self.get_frequencies(s)

        t_frequencies: list[int]=self.get_frequencies(t)

        return s_frequencies==t_frequencies

    def get_frequencies(self, string: str) -> list[int]:
        frequencies: list[int]=[0] * 123

        for letter in string:
            ascii: int=ord(letter)

            frequencies[ascii]+=1

        return frequencies

File: LeetCode/Rust/Easy/242-Valid-Anagram/solution.rs
impl Solution
{
    pub fn is_anagram(s: String, t: String) -> bool
    {
        let s_frequencies: Vec<usize>=Self::get_frequencies(&s);

        let t_frequencies: Vec<usize>=Self::get_frequencies(&t);

        s_frequencies==t_frequencies
    }

    pub fn get_frequencies(string: &String) -> Vec<usize>
    {
        let mut frequencies: Vec<usize>=vec![0; 26];

        for character in string.chars()
        {
            let ascii: usize=character as usize - 'a' as usize;

            frequencies[ascii]+=1;
        }

        frequencies
    }
}